lib/database/handler.go

```diff
@@ -11,60 +10,76 @@ import (
 	"github.com/sdslabs/gasper/types"
 )
 
+type containerHandler struct {
+	dockerImage      string
+	port             string
+	env              types.M
+	workdir          string
+	storedir         string
+	containerID, err func(string, string, string, string, types.M, string) (string, error)
+}
+
+var storepath, _ = os.Getwd()
+
+var databaseMap = map[string]*containerHandler{
+	types.MongoDB: {
+		dockerImage: configs.ImageConfig.Mongodb,
+		port:        strconv.Itoa(configs.ServiceConfig.Kaen.MongoDB.ContainerPort),
+		env:         configs.ServiceConfig.Kaen.MongoDB.Env,
+		workdir:     "/data/db",
+		storedir:    filepath.Join(storepath, "mongodb-storage"),
+		containerID: docker.CreateMongoDBContainer,
+	},
+	types.MongoDBGasper: {
+		dockerImage: configs.ImageConfig.Mongodb,
+		port:        strconv.Itoa(configs.ServiceConfig.Kaze.MongoDB.ContainerPort),
+		env:         configs.ServiceConfig.Kaze.MongoDB.Env,
+		workdir:     "/data/db",
+		storedir:    filepath.Join(storepath, "gasper-mongodb-storage"),
+		containerID: docker.CreateMongoDBContainer,
+	},
+	types.MySQL: {
+		dockerImage: configs.ImageConfig.Mysql,
+		port:        strconv.Itoa(configs.ServiceConfig.Kaen.MySQL.ContainerPort),
+		env:         configs.ServiceConfig.Kaen.MySQL.Env,
+		workdir:     "/app",
```

Comment:
workdir should be `/var/lib/mysql`

`/app` will be the workdir for the new image not this one